### PR TITLE
update

### PR DESCRIPTION
**Pull request by lordpepillo**
**Changes proposed:**

-  
-  
-  

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,16 +15,14 @@
 
 
 
-## Build Status
-
-[![master Build Status](https://travis-ci.org/AshamaneProject/AshamaneCore.svg?branch=master)](https://travis-ci.org/AshamaneProject/AshamaneCore)
-[![Build status](https://ci.appveyor.com/api/projects/status/kb0wjxh3va9c3e0k/branch/master?svg=true)](https://ci.appveyor.com/project/Traesh/ashamanecore/branch/master)
+## Build Status 
+[![master Build Status](https://travis-ci.org/Daniel25/AshamaneCore.svg?branch=master)](https://travis-ci.org/github/Daniel25/AshamaneCore)
```
Comment (DuelistRag3) [inline README.md:19]: When you do a pull request to the Master Repo, try to don´t request your readme changes.

Comment (Daniel25) [reply README.md:19]: That happens when someone create a pull request of a repository which isnt his own xD

Comment (DuelistRag3) [reply README.md:19]: Oh xD wasn´t aware of this, just quickly checked the code, for usefull stuff XD

Comment (Daniel25) [reply README.md:19]: > Oh xD wasn´t aware of this, just quickly checked the code, for usefull stuff XD

It's ok,not much usefull stuff there xD

**Pull request closed without merging**